docs/antctl.md

```diff
@@ -12,14 +12,16 @@ running in two different modes:
 
 ## Table of Contents
 
-- [Installation](#installation)
-- [Usage](#usage)
-  - [Collecting support information](#collecting-support-information)
-  - [`controllerinfo` and `agentinfo` commands](#controllerinfo-and-agentinfo-commands)
-  - [NetworkPolicy commands](#networkpolicy-commands)
-  - [Dumping Pod network interface information](#dumping-pod-network-interface-information)
-  - [Dumping OVS flows](#dumping-ovs-flows)
-  - [OVS packet tracing](#ovs-packet-tracing)
+- [Antctl](#antctl)
```

Comment:
I saw you did not change the "table of contents" format for other docs, like troubleshooting.md. Then why just change this doc?

Reply:
This was an error in my editor I just noticed. My editor tried to me "smart" and create the ToC. I will fix that. Thanks for catching.